src/styles/card.scss

```diff
@@ -0,0 +1,107 @@
+@import 'stars';
+@import 'variables';
+
+body {
```

Comment:
Styling for the body should not be defined inside a component file. It should be in a global CSS file.

Reply:
Still not fixed

Reply:
Still not fixed
<img width="823" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/790e9590-f168-4da3-b5e0-ea45f4d2baff">